### PR TITLE
Watch Mode with Chrome DevTools Workspaces

### LINKED ISSUE #757: Development Chrome DevTools Workspace
I posted https://medium.com/@rafaelideleon/webpack-your-chrome-devtools-workspaces-cb9cca8d50da#.m7bm2gcmi a while back to guide people on how to let chrome read webpack's source map during watch mode.

Ideally, the guide for here would narrow down to 
- setting `devtool: 'inline-source-map'`
- Map your Chrome DevTools Workspace to the directory with your webpack build files and your source directory to `webpack:///./`

Include _gotchas_
- Large chunks that affect the overall application have a tendency of blanking out the page, causing a need to refresh. Example: if your vendor chunk is part of a 3MB file and it gets updated in watch mode, then it is likely that the page will blank out.

And maybe a reference to the link I mentioned for more in depth guide to setting up devtools since I have images and stuff. Not really necessary.

### PR DESCRIPTION
**Pull request by rafde**
closes #757

### CODE REVIEW

**Comment by rafde** [comment]
It's possible that https://github.com/webpack/webpack/pull/4137 might resolve the issue of chrome not seeing source map changes and `source-map` can be used instead of `inline-source-map`

File: content/guides/development.md [hunk 1]
```diff
@@ -59,6 +60,24 @@ serve
 
 After each compilation, you will need to manually refresh your browser to see the changes.
 
+#### Watch Mode with Chrome DevTools Workspaces
```
Comment (bebraw) [inline content/guides/development.md]: It might be a good idea to revise the header hierarchy in this article. `##`/`###` should be enough.

Comment (rafde) [inline content/guides/development.md]: I thought it would be considered a sub topic for watch, but I guess it didn't work.

**Comment by rafde** [comment]
all fixed and ready for squash merging

**Comment by bebraw** [comment]
Nice. Thanks. 👍

**Pull request merged**